packages/synapse-react-client/src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -14,7 +14,8 @@ import {
   ACTAccessRequirement,
   EntityHeader,
   ManagedACTAccessRequirement,
-  RequestInterface,
+  Request,
+  Renewal,
```

Comment:
Much easier in TypeScript to use these explicit types than `RequestInterface` which both of these implement